cmd/sovereignnode/config/sovereignConfig.toml

```diff
@@ -37,3 +37,7 @@
     SubscribedEvents = [
     { Identifier = "bridgeOps", Addresses = ["erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"] }
 ]
+
+[OutGoingBridge]
+    GRPCHost = "localhost"
+    GRPCPort = "8085"
```

Comment:
As stated, let's not open another port. Please find a solution to this.

Reply:
This is the port of the server. Each node will be client. No port is opened on node side.